service/controller/app/v1/resource/chart/current.go

```diff
@@ -18,7 +18,7 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	name := key.AppName(cr)
+	name := cr.ObjectMeta.Name
```

Comment:
```suggestion
	name := cr.GetName()
```

Reply:
Good catch I've updated to use the functions whenever accessing the ObjectMeta struct.